test/ffmpeg-vaapi/util.py

```diff
@@ -90,6 +90,18 @@ def map_deinterlace_method(method):
     "motion-compensated": "motion_compensated",
   }.get(method, None)
 
+@memoize
+def map_transpose_direction(degrees, method):
+  return {
+    (270,   "vertical")   : "cclock_flip",
+    ( 90,            0)   : "clock",
+    (270,            0)   : "cclock",
+    ( 90,   "vertical")   : "clock_flip",
+    (180,            0)   : "reversal",
+    (  0, "horizontal")   : "hflip",
+    (  0,   "vertical")   : "vflip",
+  }.get( (degrees, method), None)
+
```

Comment:
I would prefer `None` instead of `0` as user config value for disabling mirror method.

Reply:
updated,  please review.